.gitlab/Dockerfile-2.7.8

```diff
@@ -0,0 +1,95 @@
+#
+# NOTE: THIS DOCKERFILE IS GENERATED VIA "apply-templates.sh"
```

Comment:
Hey, I don't see this `apply-templates.sh` anywhere -- mind adding a note/link to where it can be found?